src/Microsoft.ML.PipelineInference/TransformInference.cs

```diff
@@ -712,7 +711,7 @@ public override IEnumerable<SuggestedTransform> Apply(IntermediateColumn[] colum
                         {
                             Name = columnNameQuoted.ToString(),
                             Source = columnNameQuoted.ToString(),
-                            ResultType = ML.Transforms.DataKind.R4
+                            ResultType = ML.Data.DataKind.R4
```

Comment:
>ResultType = ML.Data.DataKind.R4 [](start = 28, length = 32)

Huh... this is problematic. Classes in runtime referring to the entry-point derived API.